### DISCUSSION TITLE
내 정보 페이지 최근전적

### DISCUSSION THREAD

**Discussion by JoonhoLeecs** [ft-transcendece-team/PONG_PONG#56] [General]
<img width="1710" alt="Screenshot 2024-02-24 at 12 53 31 AM" src="https://github.com/organization-for-practice/PONG_PONG/assets/124791640/1d5ba3a3-deae-4759-8ef4-4aaead0b338b">
<img width="1710" alt="Screenshot 2024-02-24 at 12 53 18 AM" src="https://github.com/organization-for-practice/PONG_PONG/assets/124791640/68d0c4b0-40a3-4d9d-be79-164570258680">

이슈에 올리면 브랜치가 생겨서 일단 디스커션에 올립니다.

**Comment by hijkl2e** [comment]
해결되었으므로 닫겠습니다
